src/arch/riscv/isa.cc

```diff
@@ -267,6 +267,11 @@ ISA::ISA(const Params &p) :
     _regClasses.push_back(&ccRegClass);
     _regClasses.push_back(&miscRegClass);
 
+    panic_if( p.vlen < p.elen,
```

Comment:
This should be a `fatal` (sorry to be pedantic). `fatal` means a config error, `panic` means a gem5 bug. This is a configuration error, I believe.